README.md

```diff
@@ -0,0 +1,123 @@
+# Blog App
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Blog App <a name="about-project"></a>
+
+**Blog App** is a ruby on rails application which is a classic example of a blog website. The fully functional website shows posts made and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+- **[View all posts.]**
+- **[Comment on a post.]**
+- **[Like a post.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [optional] instead of using a broken live demo link, I advise you remove the live demo section for now till when your project is deployed. Nice work.

Reply:
Thank you for your feedback. I will look into the live demo link.